src/library_glfw.js

```diff
@@ -1106,13 +1130,23 @@ var LibraryGLFW = {
     GLFW.hints = GLFW.defaultHints;
     GLFW.windows = new Array()
     GLFW.active = null;
+    // from emscripten_get_device_pixel_ratio
+    GLFW.scale  = window.devicePixelRatio || 1.0;
+
 
     window.addEventListener("gamepadconnected", GLFW.onGamepadConnected, true);
     window.addEventListener("gamepaddisconnected", GLFW.onGamepadDisconnected, true);
     window.addEventListener("keydown", GLFW.onKeydown, true);
     window.addEventListener("keypress", GLFW.onKeyPress, true);
     window.addEventListener("keyup", GLFW.onKeyup, true);
     window.addEventListener("blur", GLFW.onBlur, true);
+    // from https://stackoverflow.com/a/70514686/7484780 . maybe add this to browser.js?
+    // no idea how to remove this listener.
+    (function updatePixelRatio(){
+      window.matchMedia(`(resolution: ${window.devicePixelRatio}dppx)`)
```

Comment:
We don't use string templates yet in emscripten JS code.  (the strings with backticks).  So I think its safest just to use addition here.